swagger_to/py_client.py

```diff
@@ -1525,9 +1524,14 @@ def _wrap_response(resp: requests.Response) -> HTTPResponse:
 class RemoteCaller:
     """Executes the remote calls to the server."""
 
-    def __init__(self, url_prefix: str, auth: Optional[requests.auth.AuthBase] = None) -> None:
+    def __init__(self, url_prefix: str, auth: Optional[requests.auth.AuthBase] = None, session: Optional[requests.Session] = None) -> None:
```

Comment:
Please fix the line width.

```suggestion
    def __init__(
        self, 
        url_prefix: str, 
        auth: Optional[requests.auth.AuthBase] = None, 
        session: Optional[requests.Session] = None) -> None:
```